code/modules/power/tesla/energy_ball.dm

```diff
@@ -319,18 +332,30 @@
 		closest_grounding_rod.tesla_act(power, tesla_flags, shocked_targets)
 
 	else if(closest_mob)
+		if(zap_gib && iscarbon(closest_mob))
+			var/mob/living/carbon/C = closest_mob
+			C.zapping_gib = TRUE
```

Comment:
You don't need to do this, this can be an argument on ``electrocute_act`` or ``tesla_zap``.

Reply:
it needs to check the zap_gib first

Reply:
that var can be used for future use as well for admin weapon and such

Reply:
They can also just use the arg instead, I think as a var on the mob it doesn't make much sense.

Reply:
if i want to change this i have to put an argument in electrocute_act() and change other electrocute_act() overwrite codes as well

Reply:
ye yeah yeah